code/game/machinery/ecto_sniffer.dm

```diff
@@ -0,0 +1,63 @@
+/obj/machinery/ecto_sniffer
+	name = "ectoscopic sniffer"
+	desc = "A highly sensitive parascientific instrument calibrated to detect the slightest whiff of ectoplasm."
```

Comment:
false advertising, this doesn't react to the ectoplasm item(s).

Reply:
just make it say what does it do, no one's got time for bullshit riddles

Reply:
I think  the primary job of the description is to add a little flavour. 

If after reading the description and observing the device, they still don't get what it does, they should be asking their science department coworkers. 

Learning from other people ingame is always more rewarding, fosters interactions and often allows you to pick up some neat tricks.

Reply:
flavour is ok, but it also rises the already enormous entry barrier. you have literally thousands of items on the station and guessing what they do and how to use them is very time-consuming. if anything, cryptic, non-informative descriptions lead to code-diving